frappe/public/scss/common/css_variables.scss

```diff
@@ -225,4 +225,7 @@
 	--checkbox-right-margin: var(--margin-xs);
 	--checkbox-size: 14px;
 	--checkbox-focus-shadow: 0 0 0 2px var(--gray-300);
+
+	--right-arrow-svg: url("data: image/svg+xml;utf8, <svg width='6' height='8' viewBox='0 0 6 8' fill='none' xmlns='http://www.w3.org/2000/svg'><path d='M1.25 7.5L4.75 4L1.25 0.5' stroke='%231F272E' stroke-linecap='round' stroke-linejoin='round'/></svg>");
```

Comment:
should we add these via svgs?(icon system)
https://github.com/frappe/frappe/tree/develop/frappe/public/icons

Reply:
@hasnain2808 SVG symbols are not accessible via CSS.